src/garage/torch/__init__.py

```diff
@@ -1 +1,4 @@
 """PyTorch-backed modules and algorithms."""
+from garage.torch.utils import Utils
```

Comment:
can you just export these all to `garage.torch` (i.e., here) and put them in a file called `_functions.py`?